Fabric.Authorization.IntegrationTests/Modules/ChildGroupsTests.cs

```diff
@@ -166,7 +166,7 @@ public async Task AddChildGroup_ValidRequest_SuccessAsync()
                 });
             });
 
-            Assert.Equal(HttpStatusCode.Created, postResponse.StatusCode);
+            Assert.Equal(HttpStatusCode.OK, postResponse.StatusCode);
```

Comment:
For posting, wouldnt you want Created?

Reply:
In this case, we're creating an association between 2 resources (groups), so conceptually we're updating 2 existing groups. I return a 200 with the modified parent group in the response. This is consistent with the other auth endpoints that create "association" records.

Reply:
cool, good to know.